lib/elixir/lib/exception.ex

```diff
@@ -240,7 +240,7 @@ defmodule Exception do
   defp format_exit_reason(:noconnection), do: "no connection"
 
   defp format_exit_reason(:noproc) do
-    "no process"
+    "no process: the process is dead or no process is associated with the given name, possibly because its application isn't started"
```

Comment:
Hm. How about something like:

"no process: there is no running process associated with the given name, possibly because its application isn't started, or because the process died"

Then it is perhaps clearer that the process having died is just one possible explanation. I do like how "died" more directly suggests one possible reason for the error, than something like "is not alive".

Would the above be accurate? I'm not entirely sure if you can only get `:noproc` when providing a name. Never when working with PIDs? Things like via tuples? Maybe they count as names?